src/schema/v2/homeView/sections.ts

```diff
@@ -92,7 +103,29 @@ export const HeroUnits: HomeViewSection = {
   resolver: HeroUnitsResolver,
 }
 
-const sections: HomeViewSection[] = [
+export const featuredCollection = (
```

Comment:
maybe `featuredCollectionSectionBuilder` is a better name for this outer function